buildSrc/src/main/kotlin/com/saveourtool/sarifutils/buildutils/DetektConfiguration.kt

```diff
@@ -21,6 +25,23 @@ fun Project.configureDetekt() {
         config = rootProject.files("detekt.yml")
         buildUponDefaultConfig = true
     }
+    if (path == rootProject.path) {
```

Comment:
looks like `detektAll` also should be registered

Reply:
Fixed in

https://github.com/saveourtool/sarif-utils/pull/61/commits/6b2d6b367563a65d5cfdc39de4a0c2ef07910118